docs/src/index.md

```diff
@@ -18,12 +18,17 @@ You can also watch this 8-minutes video that introduces DrWatson in JuliaCon2020
 <iframe width="560" height="400" src="https://www.youtube-nocookie.com/embed/jKATlEAu8eE" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 ```
 
-To install, simply type `] add DrWatson` in your Julia session.
 DrWatson is part of [JuliaDynamics](https://juliadynamics.github.io/JuliaDynamics/), check out our [website](https://juliadynamics.github.io/JuliaDynamics/) for more cool stuff!
 
 !!! note "Star us on GitHub!"
     If you like DrWatson the please consider starring the [GitHub repository](https://github.com/JuliaDynamics/DrWatson.jl). This gives as an accurate lower bound of the number of people the software has helped!
 
+## Installing and Updating
+To install or update DrWatson, simply type `] add DrWatson` or `] up DrWatson`, respectively, in your Julia session.
+
+!!! note "Update messages"
+    Upon update DrWatson will display a message highlighting any major changes. To disable these messages set the environment variable `DRWATSON_UPDATE_MSG=0` prior to starting your Julia session. 
```

Comment:
@agerlach is this true? Isn't it enough to set `DRWATSON_UPDATE_MSG=0` prior to `using DrWatson`? I could be after Julia starts. Right?

Reply:
Julia reads the OS env vars and populates the `ENV` dict at startup. You could set `ENV["DRWATSON_UPDATE_MSG"]=0` after starting Julia and before `using DrWatson` though.

From https://docs.julialang.org/en/v1/manual/environment-variables/

> In Bash, environment variables can either be set manually by running, e.g., export JULIA_NUM_THREADS=4 before starting Julia

> Supposing that you want to set the environment variable JULIA_EDITOR to vim, you can type ENV["JULIA_EDITOR"] = "vim" (for instance, in the REPL) to make this change on a case by case basis, or add the same to the user configuration file ~/.julia/config/startup.jl in the user's home directory to have a permanent effect.